xbmc/GUIPassword.cpp

```diff
@@ -314,9 +313,8 @@ bool CGUIPassword::CheckLock(LockType btnType, const std::string& strPassword, i
       CProfilesManager::GetInstance().GetMasterProfile().getLockMode() == LOCK_MODE_EVERYONE || g_passwordManager.bMasterUser)
     return true;
 
-  int iVerifyPasswordResult = -1;
   std::string strHeading = g_localizeStrings.Get(iHeading);
-  iVerifyPasswordResult = VerifyPassword(btnType, strPassword, strHeading);
+  int iVerifyPasswordResult = VerifyPassword(btnType, strPassword, strHeading);
```

Comment:
Shouldn't  `iVerifyPasswordResult` in `CGUIPassword::VerifyPassword` set to -1 in order to keep behavior?